src/oauth2-client/index.ts

```diff
@@ -20,7 +26,7 @@ abstract class OAuth2Client {
     const { serverConfig, clientId, redirectUri, scope } = Config.get(options);
 
     /* eslint-disable @typescript-eslint/camelcase */
-    const requestParams: any = {
+    const requestParams: AnyObject = {
```

Comment:
I don't believe this aligns with the definition of `AnyObject` as these properties have actual values, yeah?